cni/network/network.go

```diff
@@ -996,13 +1031,16 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 
 	if !nwCfg.MultiTenancy {
 		// Call into IPAM plugin to release the endpoint's addresses.
-		for _, address := range epInfo.IPAddresses {
-			logAndSendEvent(plugin, fmt.Sprintf("Release ip:%s", address.IP.String()))
-			err = plugin.ipamInvoker.Delete(&address, nwCfg, args, nwInfo.Options)
-			if err != nil {
-				return plugin.RetriableError(fmt.Errorf("failed to release address: %w", err))
-			}
+		addresses := make([]*net.IPNet, len(epInfo.IPAddresses))
+		for i := range epInfo.IPAddresses {
+			addresses[i] = &epInfo.IPAddresses[i]
+			logAndSendEvent(plugin, fmt.Sprintf("Release ip:%s", epInfo.IPAddresses[i].IP.String()))
```

Comment:
this log will result into having multiple lines indicating 
Release ip: X
Release ip: Y

but we are not releasing ip X yet. I suggest us to print a single line indicating all the IPs we are attempting to release